### ISSUE TITLE
[feature request] deployment script for AWS lambda?

### ISSUE THREAD

**Issue by PicoCreator** [uilicious/inboxkitten#2]

**Comment by codehakase** [comment]
@PicoCreator I believe you are referring to creating lambda functions and deploy scripts for the api?

**Comment by PicoCreator** [comment]
@codehakase : Yup ideally allowing it to be easily deployed in the same way as firebase - https://github.com/uilicious/inboxkitten/tree/master/deploy/firebase

Hopefully it shouldn't need much changes to support (havent used aws lambda personally)
